packages/app-accounts/src/Creator.tsx

```diff
@@ -230,17 +232,24 @@ class Creator extends React.PureComponent<Props, State> {
 
   renderModalContent () {
     const { t } = this.props;
+    const { address } = this.state;
 
     return [
       <Modal.Header key='header'>
         {t('seedWarning.header', {
-          defaultValue: 'Warning'
+          defaultValue: 'Important notice!'
         })}
       </Modal.Header>,
       <Modal.Content key='content'>
+        <p>
         {t('seedWarning.content', {
-          defaultValue: 'Before you continue, make sure you have properly backed up your seed in a safe place as it is needed to restore your account.'
+          defaultValue: 'We will provide you with a generated backup file after your account is created. Please make sure to save this file in a secure location as it is the only way to restore your account. As long as you have access to your account you can always redownload this file later.'
```

Comment:
Style nitpick - how about we separate informational text from the call to action text? Just for readability. And some padding between the message and the identicon. 
![screen shot 2018-11-13 at 9 41 05 am](https://user-images.githubusercontent.com/10432070/48432102-478e0d00-e728-11e8-9b85-5d0561b72ba1.png)

Reply:
I added the option to use  <Modal.Highlight />, it has the same properties as https://react.semantic-ui.com/elements/header/.
![screen shot 2018-11-13 at 19 35 48](https://user-images.githubusercontent.com/125398/48435238-48508e80-e77c-11e8-85e6-b8e04b736736.png)

Reply:
Hmm, that didn't work out as expected. I can't rename the semantic-ui-react modules, the build is failing because of that. I'm on it...